### PR TITLE
Add Attachments API support

### PR DESCRIPTION
**Pull request by pjanik**
This PR depends on LARA PR: https://github.com/concord-consortium/lara/pull/748
However, I've already deployed a beta version of `interactive-api-host` to NPM, so we can see how it's used in practive by other projects.

### CODE REVIEW

**Comment by pjanik** [comment]
There are two unrelated commits that remove ts-fork-checker-webpack-plugin and update GH actions setup-node to v2. I had some weird build issues related to TS errors, that's why I removed the outdated plugin (it slows down the build anyway). It helped. Then, the build was failing on GH CI - it was running out of memory. So, I tried to update the action/setup-node version and it helped.

File: js/api.ts [hunk 1]
```diff
@@ -186,6 +186,17 @@ export function fetchFirestoreJWT(classHash: string, resourceLinkId: string | nu
...
 }
 
+export function fetchFirestoreJWTWithDefaultParams(firebaseApp?: string): Promise<{token: string}> {
+  return Promise.all([fetchOfferingData(), fetchClassData()])
+    .then(([offeringData, classData]) => {
+      const resourceLinkId = offeringData.id.toString();
+      // only pass resourceLinkId if there is a studentId
+      // This could be a teacher or researcher viewing the report of a student
+      // The studentId is sent in the firestore JWT request as the target_user_id
+      return fetchFirestoreJWT(classData.class_hash, urlParam("studentId") ? resourceLinkId : null, urlParam("studentId"), firebaseApp);
```
Comment (kswenson) [inline js/api.ts:196]:
```suggestion
      const studentId = urlParam("studentId");
      return fetchFirestoreJWT(classData.class_hash, studentId ? resourceLinkId : null, studentId, firebaseApp);
```
I realize this is copy/pasted code, but I'd take this opportunity to DRY it up.

File: js/containers/report/iframe-standalone-app.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ class IframeStandaloneApp extends PureComponent {
       state = answer.get("answer");
     }
     return (
-      <InteractiveIframe src={url} state={state} style={{border: "none"}} width="100%" height="100%" />
+      <InteractiveIframe src={url} state={state} answer={answer} style={{border: "none"}} width="100%" height="100%" />
```
Comment (kswenson) [inline js/containers/report/iframe-standalone-app.js:57]: This `answer` piece seems unrelated, but I'll take your word for it that it's the right thing to do. 😉

**Review by kswenson — APPROVED** [review]
👍  LGTM!

**Review by scytacki — APPROVED** [review]
looks good to me too

File: js/api.ts (hunk 1 above)
Comment (pjanik) [reply js/api.ts]: Thanks, I've fixed it here: https://github.com/concord-consortium/portal-report/pull/353/commits/5dd86538980144e889a9743c4a9c2392fd51fe0d

File: js/containers/report/iframe-standalone-app.js (hunk 2 above)
Comment (pjanik) [reply js/containers/report/iframe-standalone-app.js:57]: This is necessary for handleGetAttachmentsAPI to work. `answer` is similar to `answerMeta` in Activity Player, it contains attachments folder and info.

**Pull request merged**